sys-devel/binutils/binutils-2.37_p1-r2.ebuild

```diff
@@ -334,6 +334,16 @@ src_install() {
 		rmdir ${d} || die
 	done
 
+        # Create relative symlinks for cross-compilation
+        if [[ ${CBUILD} != ${CTARGET} ]] ; then
+                cd ${ED}/${BINPATH}
+                for x in * ; do
```

Comment:
Can you rebase this with 'dosym -r' (from EAPI 8) instead, which does relative symlinks?